utils/utils.c

```diff
@@ -712,56 +712,66 @@ char * json_quote(char *str, int *len)
 	return p;
 }
 
+#define NEWLINE '\n'
+#define isnewline(v) (v == NEWLINE)
+
 static int setargs(char *args, char **argv)
 {
 	int count = 0;
 
 	while (*args) {
-		/* ignore spaces */
-		if (isspace(*args)) {
+		/* ignore spaces and newlines */
+		if (isspace(*args) || isnewline(*args)) {
```

Comment:
AFAIK `isspace()` covers newlines too.

Reply:
I believe you are right actually. I will revert that change